components/hybrid_engine/src/write_batch.rs

```diff
@@ -3,12 +3,13 @@
 use engine_traits::{
     KvEngine, Mutable, RegionCacheEngine, Result, WriteBatch, WriteBatchExt, WriteOptions,
 };
+use region_cache_memory_engine::RegionCacheMemoryEngine;
 
 use crate::engine::HybridEngine;
 
 pub struct HybridEngineWriteBatch<EK: KvEngine> {
-    _disk_write_batch: EK::WriteBatch,
-    // todo: region_cache_engine write batch
+    disk_write_batch: EK::WriteBatch,
+    cache_write_batch: <RegionCacheMemoryEngine as WriteBatchExt>::WriteBatch,
```

Comment:
Maybe we don't need to use the <RegionCacheMemoryEngine as WriteBatchExt>::WriteBatch, just the actual struct RegionCacheWriteBatch so that we can call set_sequence_number directly

Reply:
Done.